src/main/resources/META-INF/plugin.xml

```diff
@@ -20,9 +20,10 @@
     <!-- please see https://confluence.jetbrains.com/display/IDEADEV/Plugin+Compatibility+with+IntelliJ+Platform+Products
          on how to target different products -->
 
-    <depends>com.intellij.modules.lang</depends>
```

Comment:
Hm, I believe this is a required dependency?

Reply:
I have no idea